src/main/resources/db/migration/postgresql/V2.15.0.20240621210500__custom_ctds_more_restricted_read_restricted_role.sql

```diff
@@ -205,3 +205,8 @@ from
  join ${ohdsiSchema}.sec_permission on concat('cohortdefinition:', COHORT_DEFINITION_SEC_ROLE.cohort_definition_id, ':copy:get') = sec_permission.value
 ON CONFLICT (role_id, permission_id)
 DO NOTHING;
+
+
+-- CTDS/"team project" feature specific - keep only "admin" role assignment... i.e. remove all other
```

Comment:
This `DELETE` statement need to be a separate migration, because if this migration already ran — it won't be executed.

Reply:
correct, there is no need to execute it. I'm adding it here for completeness, because it belongs to this previous deployment. We ended up running this manually...while it could have been here.

Reply:
>We ended up running this manually
OK, as long as we track and contain this change, it will be fine.

Approved. :+1: